packages/manager/apps/web/client/app/hosting/database/order/private/hosting-database-order-private.controller.js

```diff
@@ -1,122 +1,41 @@
-    this.$filter = $filter;
+  constructor($timeout, $translate, Alerter) {
     this.$timeout = $timeout;
     this.$translate = $translate;
+    this.Alerter = Alerter;
   }
 
-  checkout() {
-    this.checkoutLoading = true;
-
-    this.checkoutOrderCart(!!this.defaultPaymentMethod, this.cartId);
+  $onInit() {
+    this.alerts = {
+      main: 'app.hosting.dashboard.database.order-private',
+    };
   }
 
-  async initDurationStep() {
-    this.durationLoading = true;
-
-    // Reset cart if there is one
-    if (this.cart) {
-      await this.resetOrderCart();
-      this.cart = undefined;
-    }
-
-    // Sort prices
-    const prices = this.catalogProduct.pricings.filter(
-      ({ interval }) => interval > 0,
+  onError(error) {
+    this.Alerter.error(
+      this.$translate.instant('hosting_database_order_private_error', {
+        message: get(error, 'data.message', error),
+      }),
+      this.alerts.main,
     );
-    this.$timeout(() => {
-      this.catalogProduct.pricings = sortBy(prices, 'price');
-      this.durationLoading = false;
-    });
-  }
-
-  getDuration(interval) {
-    return this.$filter('wucDuration')(interval.toString(), 'longDate');
+    this.scrollToTop();
   }
 
-  getVersions(product) {
-    this.catalogProduct = find(this.catalogProducts, {
-      planCode: product.planCode,
-    });
-    const engines = find(this.catalogProduct.configurations, {
-      name: 'engine',
-    });
-
-    if (!engines || !engines.values) {
-      return [];
-    }
-
-    return orderBy(
-      engines.values.sort().map((engine) => {
-        const [engineLabel, engineVersion] = engine.split('_');
-        const name = this.$translate.instant(
-          `hosting_database_order_private_version_${engineLabel}`,
-          { version: engineVersion },
-        );
-
-        return {
-          name,
-          value: engine,
-          engineLabel,
-          engineVersion: parseFloat(engineVersion),
-        };
+  onSuccess(url) {
+    this.goBack(
+      this.$translate.instant('hosting_database_order_private_success', {
+        orderUrl: url,
       }),
-      ['engineLabel', 'engineVersion'],
-      ['asc', 'desc'],
     );
   }
 
-  async prepareCheckout() {
-    if (!this.cart && !this.checkoutLoading) {
-      this.checkoutLoading = true;
-      const { cart, cartId } = await this.prepareOrderCart(
-        this.order.datacenter,
-        this.order.price,
-        this.order.product,
-        this.order.version.value,
-      );
-
-      this.$timeout(() => {
-        this.cart = cart;
-        this.cartId = cartId;
-        this.checkoutLoading = false;
-      });
-    }
-  }
-
-  async updateDatacenter(serviceName) {
-    this.durationLoading = true;
-    const datacenter = await this.getDatacenter(serviceName);
-
+  scrollToTop() {
     this.$timeout(() => {
-      this.order.datacenter = datacenter;
-      this.durationLoading = false;
+      document.getElementById('order-private-database').scrollIntoView(true);
```

Comment:
You can use `$anchorScroll` -> https://docs.angularjs.org/api/ng/service/$anchorScroll